e2e-tests/cypress/tests/cypress/e2e/User.Authentication.ResetPassword.feature

```diff
@@ -0,0 +1,25 @@
+Feature: User Authentication - reset password
+  As a user
+  I want the option to reset my password from the sign in page
+
+  # TODO for these pre-conditions utilize seeding or API check, if user exists in test system
+  # Background:
+  #   Given the following "users" are in the database:
+  #     | email            | password | name          |
+  #     | bibi@bloxberg.de | Aa12345_ | Bibi Bloxberg |
+
+  Scenario: Reset password from signin page successfully
+    Given the browser navigates to page "/login"
```

Comment:
why is it the browser and not the user in the first step?

Reply:
Technically it is not an emulated user action driven by Cypress, but the browser navigating to the given URL.
For a more coherent reading of the scenario's story we can change it to the user.